README.md

```diff
@@ -88,27 +88,24 @@ usage: Options are as follows...
     sumstats=TRUE
     obsLimit=1000
 
+ -- format [2.5 | 3.2]
```

Comment:
Is there an extra space before format? Should this be --format, not -- format? Also maybe have an extra blank line after this line, to be consistent with the existing formatting of the file. e.g.

--format [2.5 | 3.2]
<this is a new blank line>
--exclude <arg> (optional) exclude statistics for variables specified in the file with specified path. Format for the exlude file: